src/main/groovy/com/getkeepsafe/dexcount/System.groovy

```diff
@@ -0,0 +1,38 @@
+package com.getkeepsafe.dexcount
```

Comment:
It *should*, yes; as long as it isn't in the same package as the task or higher, it won't cause problems elsewhere.